### ISSUE TITLE
Cursor in search box became invisiable in Moon theme

### ISSUE THREAD

**Issue by MkQtS** [Kunzisoft/KeePassDX#1402]
**Describe the bug**

It seems that no matter what theme is used, the cursor after text in search box is blue. This causes an issue that the cursor becomes invisible in *Moon* theme.

**To Reproduce**

1. Choose the *Moon* theme
2. Open a kdbx file
3. Type in the search box
4. The cursor in search box is invisiable

**Expected behavior**

White cursor is shown in the search box.

**KeePassDX:**

 - Version: 3.4.5
 - Build: F-droid
 - Language: Simplified Chinese

**Android:**

 - Device: POCO F2 Pro
 - Version: Android 12

**Comment by J-Jamet** [comment]
![Screenshot from 2022-09-15 15-43-38](https://user-images.githubusercontent.com/19224038/190420059-cde6e51b-d5c5-4712-b7d2-1a42196b13cd.jpg)

Just checked and the cursor is well visible in Android 12
The blue of the cursor is lighter.

**Comment by MkQtS** [comment]
I just updated to 3.5.0 Beta01 on F-droid and taked a screenshot

![image](https://user-images.githubusercontent.com/81752398/190430803-9c8a4a7b-5875-459d-a6f1-87be4707b67f.png)

So this may be a system related problem?

**Comment by J-Jamet** [comment]
Yes probably, it seems to use a blue that is not even in the theme. Must be a problem with the OS POCO overlay but I can't check so it will be difficult to know the exact reason.
